package ExceptionHandling;

import java.util.Scanner;

public class Main 
{
	public static void main(String[] args) 
	{
		int a, b, c;
		
		try
		{
			Scanner user = new Scanner(System.in);
			int b = user.nextInt();
			
			int a = c / b;
			
			System.out.println("working Code");
		}
		catch(ArithmeticException e)
		{
			System.out.println("you broke it");
		}
		catch(Exception e)
		{
			System.out.println("you really broke it");
		}
		finally
		{
			System.out.println("hello");
		}
	}
}
